metadata/oswitch.xml

```diff
@@ -24,5 +24,25 @@
 			<_long>Moves focus to the previous output with the focused window with the specified activator.</_long>
 			<default></default>
 		</option>
+		<option name="left_output" type="activator">
+			<_short>Left output</_short>
+			<_long>Moves focus to the output on the left.</_long>
+			<default>&lt;super&lt; Left</default>
```

Comment:
I think we should disable this by default (leave empty or `disabled` works too), when we add new bindings, they might conflict with existing bindings from users.